packages/trpc/server/routers/viewer/organizations/update.schema.ts

```diff
@@ -22,6 +25,7 @@ export const ZUpdateInputSchema = z.object({
   timeZone: z.string().optional(),
   weekStart: z.string().optional(),
   timeFormat: z.number().optional(),
+  metadata: teamMetadataSchema.unwrap().pick({ isOrganizationConfigured: true }),
```

Comment:
@zomars here is the restriction to only have `isOrganizationConfigured` be passed down to be updated